### PR TITLE
avoids 500 error when invalid owner name is provided

### PR DESCRIPTION
**Pull request by shamsimam**
## Changes proposed in this PR

- avoids 500 error when invalid owner name is provided

## Why are we making these changes?

An invalid owner in the query string should not result in a server error, instead should return an empty result.

### CODE REVIEW

File: waiter/src/waiter/token.clj [hunk 1]
```diff
@@ -201,9 +201,9 @@
     "List all tokens for a given user."
     [kv-store owner]
     (let [owner->owner-key (kv/fetch kv-store token-owners-key)]
-      (if-let [owner-key (owner->owner-key owner)]
+      (if-let [owner-key (get owner->owner-key owner)]
```
Comment (dposada) [inline waiter/src/waiter/token.clj]: Why was this change needed?

Comment (shamsimam) [reply waiter/src/waiter/token.clj]: Because `owner->owner-key` is `nil` in some cases (e.g. waiter just started up with an no tokens)
